template/.postcssrc.js

```diff
@@ -0,0 +1,16 @@
+const postcssConfig = {
+  plugins: [require('tailwindcss'), require('autoprefixer')],
+};
+
+if (process.env.NODE_ENV === 'production') {
+  postcssConfig.plugins.push(
+    require('@fullhuman/postcss-purgecss')({
+      content: [
+        './web/wp-content/themes/fqdn/src/js/**/*.js',
+        './web/wp-content/themes/fqdn/templates/**/*.twig',
```

Comment:
Should this be a dynamic value ? (replaced upon creating with the script ?)

Reply:
Nice catch :)